backend/users/Users.ts

```diff
@@ -0,0 +1,32 @@
+import { sequelize } from "../DB.js";
+
+export async function findOrCreateLocalUser(email) {
+  const users = await sequelize.models.User.findAll({
+    where: {
+      email: email,
+    },
+  });
+
+  let localUser = users.length > 0 ? users[0] : null;
+
+  if (!localUser) {
+    localUser = await sequelize.models.User.create({
+      firstName: "localDevFirstName",
+      lastName: "localDevLastName",
+      email: email,
+      password: "localDevPassword",
+      googleAuthToken: "localDevGoogleAuthToken",
```

Comment:
I think I'd prefer `null` for the googleAuthToken, since otherwise our code will think that the local user has a google account but then will fail when it calls the google apis with `localDevGoogleAuthToken` rather than a real auth token.